release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3311,23 +3311,76 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, builds could not set the `GIT_LFS_SKIP_SMUDGE` environment variable and use its value when cloning source code. This caused builds to fail for some git repositories with LFS files. With this release, the build is allowed to set this environment variable and use it during the git clone step of the build. (link:https://issues.redhat.com/browse/OCPBUGS-35283[*OCPBUGS-35283*])
+
+* Previously, registry overrides were present in non-relevant data plane images. With this release, the way {product-title} propagates the override-registries has been modified and the issue is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-34602[*OCPBUGS-34602*])
+
+* Previously, RegistryMirrorProvider images were not being updated during the reconciliation because RegistryMirrorProvider was modifying the cached image directly instead of the internal entries. With this release, the way we update the images has been modified, avoiding the cache and doing it directly in the entries so the bug no longer presents. (link:https://issues.redhat.com/browse/OCPBUGS-34569[*OCPBUGS-34569*])
+
+* Previously, the `alertmanager-trusted-ca-bundle` configmap was not injected into the user-defined Alertmanager container, which prevented the verification of HTTPS web servers receiving alert notifications. With this update, the trusted CA bundle configmap is mounted into the Alertmanager container at the `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem` path. (link:https://issues.redhat.com/browse/OCPBUGS-34530[*OCPBUGS-34530*])
+
+* Previously, in Amazon Web Services (AWS) security token service (STS), Cloud Credential Operator (CCO) checked `awsSTSIAMRoleARN` in `CredentialsRequest` to create the secret. When awsSTSIAMRoleARN was not present, CCO logged the error. With this release, CCO no longer logs the error and the bug is fixed. (link:https://issues.redhat.com/browse/OCPBUGS-34117[*OCPBUGS-34117*])
+
+* Previously, with the OVN-Kubernetes setting for routing-via-host set to shared gateway mode, its default value, OVN-Kubernetes did not correctly handle traffic streams that mixed non-fragmented and fragmented packets from the IP layer on cluster ingress. This caused connection resets or packet drops. With this release, OVN-Kubernetes correctly reassembles and handles external traffic IP packet fragments on ingress. (link:https://issues.redhat.com/browse/OCPBUGS-29511[*OCPBUGS-29511*])
+
+[id="ocp-4-16-2-known-issue_{context}"]
+==== Known issue
+
+* If the `ConfigMap` maximum transmission unit (MTU) is absent in the namespace 'openshift-network-operator', users have to create the `ConfigMap` manually with the machine MTU value, before starting the live migration. Otherwise, the live migration will get stuck and fail. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])
```

Comment:
```suggestion
* If the `ConfigMap` maximum transmission unit (MTU) is absent in the namespace `openshift-network-operator`, users have to create the `ConfigMap` manually with the machine MTU value, before starting the live migration. Otherwise, the live migration will get stuck and fail. (link:https://issues.redhat.com/browse/OCPBUGS-35829[*OCPBUGS-35829*])
```